Chummer/data/gear.xml

```diff
@@ -1532,6 +1532,68 @@
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
     </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>6P Acid</damagereplace>
+      </weaponbonus>
+      <ammoforweapontype>slingshot</ammoforweapontype>
+    </gear>
+	<gear>
+      <id>e20f021b-1d00-4443-a481-d417114e067e</id>
+      <name>Ammo: Sling-Shot Paint Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>10</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>0</damagereplace>
```

Comment:
If you are doing a `damagereplace`, you should specify the type of damage, just to be on the safe side. That is why the T-Shirt Cannon has a damage code of `0S` instead of `0`.